thermalprinter/thermalprinter.py

```diff
@@ -586,9 +587,12 @@ def sleep(self, seconds=1):
             self.__is_sleeping = True
         self.send_command(Command.ESC, 56, seconds, seconds >> 8)
 
-    def status(self):
+    def status(self, suppress_communication_error=True):
```

Comment:
I would rather name the argument `raise_on_error` with default value set as `True`. The behavior is more logical to ask for quiet rather than ask for verborsity.